src/main/java/webserver/HttpRequest.java

```diff
@@ -0,0 +1,80 @@
+package webserver;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import utils.IOUtils;
+
+public class HttpRequest {
+    public static final int KEY_INDEX = 0;
+    public static final int VALUE_INDEX = 1;
+
+    private final HttpRequestMethod httpRequestMethod;
+    private final HttpRequestFirstLine httpRequestFirstLine;
+    private final HttpRequestHeaders httpRequestHeaders;
+    private final HttpRequestBody httpRequestBody;
+
+    private HttpRequest(HttpRequestMethod httpRequestMethod, HttpRequestFirstLine httpRequestFirstLine, HttpRequestHeaders httpRequestHeaders, HttpRequestBody httpRequestBody) {
+        this.httpRequestMethod = httpRequestMethod;
+        this.httpRequestFirstLine = httpRequestFirstLine;
+        this.httpRequestHeaders = httpRequestHeaders;
+        this.httpRequestBody = httpRequestBody;
+    }
+
+    public static HttpRequest of(BufferedReader bufferedReader) throws IOException {
+        String line = bufferedReader.readLine();
+
+        if (line == null) {return null;}
```

Comment:
IDE 자동정렬을 진행했는데도 지금처럼 정렬이되시던가요? 🤔

아래와 같이 코드 라인이 한줄이더라도 `{ }`은 줄바꿈을해서 아래와 같이 변경하는 것을 의도한거였어요. 👀
```suggestion
        if (line == null) {
            return null;
        }
```